lib/std/heap.zig

```diff
@@ -325,8 +325,9 @@ pub fn pageSize() usize {
 // A cache used by `defaultQueryPageSize()` to avoid repeating syscalls.
 var page_size_cache = std.atomic.Value(usize).init(0);
 
-// Default querying function for `pageSize()`.
-pub fn defaultQueryPageSize() usize {
+// The default implementation in `std.options.queryPageSizeFn`.
+// The first time it is called, it asserts that the page size is within the comptime bounds.
+pub fn defaultQueryPageSizeFn() usize {
```

Comment:
`defaultQueryPageSize` was fine - the `Fn` suffix is usually only used for function values/pointers. See e.g. `std.Options.logFn` vs `std.log.defaultLog`.